addons/project_mail_plugin/views/project_task_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- action used by the mail plugins in order to redirect the user to the newly created task in edit mode-->
+    <record id="task_project_action_form_edit" model="ir.actions.act_window">
```

Comment:
Rename to: "project_task_action_form_edit" to comply with what has been done in helpdesk.

Reply:
Changed to `project_task_action_form_edit` @std-odoo this will require a change in the gmail part